### PR TITLE
Updated and fixed some bugs

### PR DESCRIPTION
**Pull request by MechanicKim**
1. Changed splash image(Android)
2. Changed Logo in Login and SignUp page.
3. Changed some files' name(FindPassword1 ~ 3)
4. Fixed some bugs about sign in, sign up and all processes of reset password.

### CODE REVIEW

File: src/utils/ApiMeta.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ export const ServerMeta = {
 
 export const UrlMeta = {
   HOST: 'http://ec2-52-78-121-221.ap-northeast-2.compute.amazonaws.com:80/',
+  //HOST: 'http://192.168.0.53:8000/',  // test url
```
Comment (SarahhhC) [inline src/utils/ApiMeta.js]: Delete this.

File: src/utils/ApiUtil.js [hunk 2]
```diff
@@ -35,17 +35,13 @@ class ApiUtil {
   requestGetWithTokenUrl(callback, api, etcUrl, body = null,
     contentType = ServerMeta.CONTENT_TYPE_URL) {
     AsyncStorage.getItem('token', (err, result) => {
-      if (err) return;
-      if (result)
-       this.request(callback, api, 'GET', body, contentType, result, etcUrl);
+      this.request(callback, api, 'GET', body, contentType, result, etcUrl);
```
Comment (SarahhhC) [inline src/utils/ApiUtil.js]: Why don't you handle err? If this part always occur err, then you should `if (err) { ... }`

File: src/utils/ApiUtil.js [hunk 3]
```diff
@@ -35,17 +35,13 @@ class ApiUtil {
...
     });
   }
 
   requestPostWithToken(callback, api, body, contentType = ServerMeta.CONTENT_TYPE_JSON)  {
     AsyncStorage.getItem('token', (err, result) => {
-      if (err) return;
-      if (result)
-       this.request(callback, api, 'POST', body, contentType, result);
+      this.request(callback, api, 'POST', body, contentType, result);
```
Comment (SarahhhC) [inline src/utils/ApiUtil.js]: ditto

File: src/components/Login/Login.js [hunk 4]
```diff
@@ -25,29 +25,47 @@ class Login extends Component {
...
+
+  hasToken() {
     AsyncStorage.getItem('token', (err, result) => {
-      if (result)
+      if (result) {
         UserUtil.getMyProfile(this.onTokenValidCheck.bind(this));
-      else
+      } else {
         this.setState({ loaded: true });
+      }
```
Comment (heartjigi) [inline src/components/Login/Login.js:37]: What about
if (err) {
  this.setState({ loaded: true });
} else if (result) {
  UserUtil.getMyProfile(this.onTokenValidCheck.bind(this));
}

Comment (MechanicKim) [reply src/components/Login/Login.js:37]: Oh! I got it! Thanks!

File: src/components/Login/SignUp.js [hunk 5]
```diff
@@ -25,40 +25,25 @@ class SignUp extends Component {
...
-
-    let onChangePassword1 = (text) => { this.state.password1 = text; };
-
-    let onChangePassword2 = (text) => { this.state.password2 = text; };
-
-    let signInWithFacebook = () => UserUtil.signInWithFacebook(this.onSignUpCallback.bind(this));
-
+    let onChangeEmail = (text) => this.state.email = text;
+    let onChangePassword1 = (text) => this.state.password1 = text;
+    let onChangePassword2 = (text) => this.state.password2 = text;
```
Comment (heartjigi) [inline src/components/Login/SignUp.js]: why don't we change 
onChangePassword1 -> InputEmailAddr
onChangePassword2 -> InputSecretCode

File: src/components/Login/Styles.js [hunk 6]
```diff
@@ -13,9 +13,16 @@ const styles = StyleSheet.create({
     justifyContent: 'center',
     alignItems: 'center',
   },
+  mainLogoText: {
+    fontFamily: 'ProductSans-Bold',
```
Comment (heartjigi) [inline src/components/Login/Styles.js:17]: I think that we need ttf file also for Xcode, 
and we also add splash to xcode, 
I add after this PR passed

File: src/utils/UserUtil.js [hunk 7]
```diff
@@ -38,12 +38,18 @@ class UserUtil {
...
+    body.password = password;
+    apiUtil.requestPost(callback, 'API_LOCAL_SIGNIN', body);
+  }
+
   localSignUp(callback, email, password) {
     let body = {};
     body.email = email;
     body.password = password;
-    apiUtil.requestPostWithToken(callback, 'API_LOCAL_SIGNUP', body);
+    apiUtil.requestPost(callback, 'API_LOCAL_SIGNUP', body);
```
Comment (heartjigi) [inline src/utils/UserUtil.js:52]: Im sorry, It is my mistake

File: src/components/Login/SignUp.js (hunk 5 above)
Comment (MechanicKim) [reply src/components/Login/SignUp.js]: Oh! email address is okay!
password is not secret code. password1 is password ^^
and password2 is password for confirmation. So, we can change it like this.
onChangePassword1 -> onInputPassword
onChangePassword2 -> onInputConfirmPassword
or
onChangePassword1 -> onInputPassword1
onChangePassword2 -> onInputPassword2

^^;

File: src/components/Login/Styles.js (hunk 6 above)
Comment (MechanicKim) [reply src/components/Login/Styles.js:17]: I will find and add this file.
But, I can't add splash to xcode.

Thanks!

File: src/components/Login/SignUp.js (hunk 5 above)
Comment (heartjigi) [reply src/components/Login/SignUp.js]: Oh my mistake kk

File: src/components/Login/Styles.js (hunk 6 above)
Comment (heartjigi) [reply src/components/Login/Styles.js:17]: Yeah i'll handle tmr

**Comment by SarahhhC** [comment]
Good LGTM @MechanicKim 👍

**Comment by SarahhhC** [comment]
LGTM

File: src/components/Login/InputEmailAddr.js [hunk 8]
```diff
@@ -46,19 +46,20 @@ class FindPassStep1 extends Component {
 
   onRequestSecretCodeCallback(result, error) {
     if (error) {
+      console.log(error);
```
Comment (ace0625) [inline src/components/Login/InputEmailAddr.js:49]: seems you forget to delete this.

**Pull request merged**